web/cobrands/bexley/js.js

```diff
@@ -128,5 +128,27 @@ fixmystreet.assets.add(labeled_defaults, {
     asset_item: 'street light'
 });
 
+// We need to trigger the below function on subcategory change also
+$(function(){
+    $("#problem_form").on("change.category", "#form_DALocation", function() {
+        $(fixmystreet).trigger('report_new:category_change', [ $('#form_category') ]);
+    });
+});
+
+fixmystreet.message_controller.register_category({
+    body: defaults.body,
+    category: function() {
+        var cat = $('#form_category').val();
+        if (cat === 'Dead animal') {
+            var where = $('#form_DALocation').val();
+            if (where === 'Garden' || where === 'Other private property') {
+                return true;
+            }
```

Comment:
Once you select one of these options there's no way to change it without clicking the map again which is a bit annoying.

Reply:
Isn't the category box still visible too? I guess I didn't want someone just changing the subcategory in order to get it through - though you raise an important point that the link it mentions won't be visible any more, whoops, so it should somehow leave it visible..!